extensions/wikia/Insights/PagesWithoutInfobox.setup.php

```diff
@@ -4,7 +4,7 @@
  * Insights - Pages without an infobox
  *
  * CAUTION
- * BELOW CLASSES HAVE DEPENDENCIES ON CLASSES FROM Insights.setup.php - remember to load them as well
+ * BELOW CLASSES HAVE DEPENDENCIES ON CLASSES FROM InsightsV2.setup.php - remember to load them as well
```

Comment:
All changes within Insights directory wasn't intended, right? (Insights to InsightsV2 in comments in setup files)